src/localization/en/navigation.ts

```diff
@@ -26,6 +26,8 @@ export default {
         news: 'THI News',
         library: 'Library',
         libraryCode: 'Library Number',
+        licenses: 'Licenses',
```

Comment:
`Licenses` ist defined in three different localization files, can this be simplified?